.github/workflows/crate-deploy.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy crate
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "publish osm2lanes"
+        run: |
+          rustup toolchain install stable
```

Comment:
Do we not need `hecrj/setup-rust-action@v1`? How is `rustup` already around?

Reply:
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#rust-tools

Reply:
Woo, nice!